docs/source/tutorials.rst

```diff
@@ -105,43 +107,273 @@ After installing the AS2FM packages as described in the :ref:`installation secti
 The output is a JANI file called `main.jani` that will be located in the same folder.
 
 
-.. _jani_conversion:
 
-How to convert from CONVINCE robotic JANI to plain JANI?
------------------------------------------------------------
 
-.. note::
-    The CONVINCE robotic JANI extension is not related to the modeling of robotics system using ROS, unlike the SCXML approach. The following tutorial targets users that want to develop a JANI model of a robotics system by hand.
 
-We provide a Python script to convert models describing the system and its environment together, given in the CONVINCE robotics JANI flavor as specified in the `data model repository <https://github.com/convince-project/data-model>`_, into `plain JANI <https://jani-spec.org>`_ accepted as input by model checkers.
 
```

Comment:
Avoid more than one empty line.

I think we should add

```
- repo: https://github.com/rstcheck/rstcheck
  rev: main # should be replaced with the current verison
  hooks:
    - id: rstcheck
      additional_dependencies: ['rstcheck[sphinx,toml]']
```

I assume this would find that, but I didn't check.